pkg/storage/shipper/shipper.go

```diff
@@ -3,11 +3,12 @@
 // Provenance-includes-license: Apache-2.0
 // Provenance-includes-copyright: The Thanos Authors.
 
-package ingester
+package shipper
```

Comment:
> Extracting `shipper` from ingester to a dedicated package

If we want to go this route, I suggest we do that separately, landing this to the `main` branch first. Otherwise, it will be fairy hard to merge `blockbuilder` back to `main` later